sixMans/sixMans.py

```diff
@@ -437,11 +437,15 @@ async def lobbyInfo(self, ctx: Context):
         game = self._get_game_by_text_channel(ctx.channel)
         player = ctx.message.author
         if game:
-            await game.post_lobby_info()
-            await ctx.message.add_reaction(Strings.WHITE_CHECK_REACT)
+            if game.has_lobby_info():
+                await game.post_lobby_info()
+                await ctx.message.add_reaction(Strings.WHITE_CHECK_REACT)
+            else:
+                await ctx.message.add_reaction(Strings.WHITE_X_REACT)
+                await ctx.send(f"{player.mention}, the game is not ready yet.")
```

Comment:
I think we could probably ignore the command when the lobby info isn't ready.

Reply:
With this change, the logic to check to see if the lobby info is ready could just be moved into the `post_lobby_info` function.

Reply:
updated